scripts/build/with-react.js

```diff
@@ -7,6 +7,7 @@ if (typeof window !== 'undefined') {
 	window.AlloyEditor = AlloyEditor;
 	AlloyEditor.React = React;
 	AlloyEditor.React.PropTypes = PropTypes;
+	AlloyEditor.version = require('../../package.json').version;
```

Comment:
This is good but I'd go with `VERSION` to make it clear that this is a "constant".